extensions/azurePublish/src/node/deploy.ts

```diff
@@ -43,9 +47,19 @@ export class BotProjectDeploy {
     name: string,
     environment: string,
     hostname?: string,
-    luisResource?: string
+    luisResource?: string,
+    absSettings?: any
   ) {
     try {
+      console.log(absSettings);
+      if (absSettings) {
+        await this.linkBotWithWebapp(absSettings, hostname);
```

Comment:
@zidaneymar don't make it abs specific, we should do this as long as the subscriptionid, resourceGroup and botName is available, let's follow up on this offline.